spacy_transformers/layers/hf_shim.py

```diff
@@ -0,0 +1,95 @@
+from io import BytesIO
+from pathlib import Path
+from typing import Any
+import srsly
+from functools import partial
+import torch
+from spacy.vectors import get_current_ops
+
+from ..util import make_tempdir
+
+from thinc.api import PyTorchShim
+
+from transformers import AutoModel, AutoConfig, AutoTokenizer
+
+
+class HFModel:
```

Comment:
Would `HFObjects` be a better name for this collection?

There's nothing wrong with your code, but was there a reason to not use a dataclass here?

Reply:
You're right, and using `default_factory` will also prevent the default dict issue.